python/dgl/distributed/rpc_client.py

```diff
@@ -165,12 +165,18 @@ def connect_to_server(ip_config, num_servers, max_queue_size=MAX_QUEUE_SIZE,
     rpc.create_sender(max_queue_size, net_type)
     rpc.create_receiver(max_queue_size, net_type)
     # Get connected with all server nodes
+    max_try_times = int(os.environ.get('DGL_DIST_MAX_TRY_TIMES', 1024))
```

Comment:
On L177, there is a 3 sec interval between two retries. Meaning the max timeout will be ~1 hour. Users may still be confused by this. Consider setting it to be 5~10 minutes (e.g., 100~200 retries).

Reply:
Timeout in `rpc.connect_receiver` and `rpc.connect_receiver_finalize` are mutually exclusive. The former one  takes effect for `tensorpipe` only, the latter one takes effect for `socket` backend only. It looks confusing.

Reply:
Understand that there are two different retry logics. My concern is that the retry times is too large, meaning there will be long silence till a timeout error. Users will probably think the program is hanging.

Reply:
printing some log continuously when re-trying may be better. `10min` may be too smaller for a large graph. I choose `1024` here because the original value is `1000` in `socket` backend. But I'm ok with changing to `10min` here.

Reply:
+1 on some log print. What will affect the server launching time? Why larger graph will be an issue?

Reply:
Loading a large graph may take long time and this happens before rpc involves in.